link.go

```diff
@@ -73,6 +78,12 @@ func (link *ToxicLink) Start(name string, source io.Reader, dest io.WriteCloser)
 				"err":   err,
 			}).Warn("Source terminated")
 		}
+		metrics.ReceivedBytesTotal.WithLabelValues(
+			streamDirection,
+			proxyName,
+			listenAddr,
+			upstreamAddr,
```

Comment:
@neufeldtech I prefer to share variables with arguments instead of closure. 
It would help to split method in future.

Abstract example:
```
go func(streamDirection, proxyName, ...) {
}(streamDirection, proxyName, ...)
```

Reply:
> It would help to have initialization part for metrics, than could be configured.

I'm glad you brought this up for discussion. I will refactor to make metrics optional (feature flagged) via a command line arg (and likely an environment variable as well, for easy Kubernetes compatibility).